nistats/tests/test_first_level_model.py

```diff
@@ -32,45 +32,18 @@
                                        run_glm,
                                        )
 
-
-from nistats.tests.test_utils import create_fake_bids_dataset
-from nistats.utils import get_bids_files
-
+from nistats.utils import (get_bids_files,
+                           )
```

Comment:
Maybe I'm wrong but I really prefer the more natural
```
from nistats.utils import get_bids file
```

Reply:
That is an oversight, the parentheses are not desired for a single entry.